src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp

```diff
@@ -180,12 +225,32 @@ struct InitializeElement {
         dt_variables_tag,
         Tags::HistoryEvolvedVariables<variables_tag, dt_variables_tag>>;
 
-    using compute_tags = db::AddComputeTags<
-        Tags::Time,
-        Tags::deriv<typename variables_tag::tags_list,
-                    typename System::gradients_tags,
-                    Tags::InverseJacobian<Tags::ElementMap<Dim>,
-                                          Tags::LogicalCoordinates<Dim>>>>;
+    template <typename LocalSystem, typename = std::nullptr_t>
```

Comment:
I'd just make the second template parameter a bool so we don't need to do SFINAE, which is slower to compile